### ISSUE TITLE
Auto switch

### ISSUE THREAD

**Issue by leonardoxh** [leonardoxh/race-control-tv#79]
From pre race show to race ?

**Comment by dseven** [comment]
Yes, please! Seems that one one occasion, at least, the prerace stream showed "coverage will begin shortly" after the show ended. When it was still like this a couple of minutes before the hour, I stopped that stream, and found that the race show had actually started already. I wonder how you would know when to do the switch, though... if they don't terminate the prerace stream when the show is over....

**Comment by leonardoxh** [comment]
We do have a few properties that indicate the event end date (including hour and seconds) that would be the way to do it
